lib/commands/touch.js

```diff
@@ -1,7 +1,8 @@
 const commands = {};
 
-commands.performActions = async function (actions) {  
-  return await this.winAppDriver.sendCommand('/actions', 'POST', {actions});};
```

Comment:
it makes sense to add a comment here that this method is only needed to make clicks on `-image` elements working properly

Reply:
Added